torq.q

```diff
@@ -424,19 +427,21 @@ createalias:{[logdir;filename;alias]
 // logname = base of log file
 // timestamp = optional timestamp value (e.g. .z.d, .z.p)
 // makealias = if true, will create alias files without the timestamp value
-createlog:{[logdir;logname;timestamp;suppressalias]
-	basename:(string logname),"_",(string timestamp),".log";
-	alias:$[suppressalias;"";(string logname),".log"];
-	fileredirect[logdir;"err_",basename;"err_",alias;2];
-	fileredirect[logdir;"out_",basename;"out_",alias;1];
+createlog:{[logdir;logname;timestamp;suppressalias;onelog]
+        out_prefix:"out_";
+	err_prefix:$[onelog;out_prefix;"err_"];
+	basename:string[logname],"_",string[timestamp],".log";
+        alias:$[suppressalias;"";string[logname],".log"];
+	fileredirect[logdir;out_prefix,basename;out_prefix,alias;1];
+        fileredirect[logdir;err_prefix,basename;err_prefix,alias;2];
```

Comment:
CAn you fix up the indentations here?

Reply:
I can't see the indentations on vim. take it that's normal?